### DISCUSSION TITLE
Syncing service with Syncthing

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferdium/ferdium-app#1697] [Ideas]
Please make it possible to sync your services between devices using tools like Syncthing.

Syncthing is software that will sync folders between two devices without storing the folder on a cloud.

Would be great if there was a way to easily sync the data of Ferdium between two devices using Syncthing.

**Comment by vraravam** [comment]
Syncthing can be used independently to sync folders (I use it all the time). you can do the same with Ferdium's profile folder even now. All you need to do is to choose the serverless/accountless option when signing into Ferdium for the first time. After this, the profile folder can be setup to be synced across different devices from Syncthing's web ui. Please ensure that Ferdium is closed/quit before syncing.

**Reply by trymeouteh** [reply]
> After this, the profile folder can be setup to be synced across different devices from Syncthing's web ui.

What is the folder directory?

> Please ensure that Ferdium is closed/quit before syncing.

If I do not close/quit Ferdium, what will happen, my new changes will not sync and be overwritten?

**Reply by vraravam** [reply]
If the intent is to sync the whole profile folder, then syncing while the app is open can cause corruption. I haven't tried out for the app that I use syncthing for.

**Comment by SpecialAro** [comment]
I'm closing this discussion as well. Let's focus this topic in: https://github.com/ferdium/ferdium-app/issues/1710
